source/_addons/configurator.markdown

```diff
@@ -62,22 +62,72 @@ Screenshot of the HASS Configurator.
 }
 ```
 
-- **username** (*Required*): Set a username to access your configuration is protected.
-- **password** (*Required*): Set a password for access.
-- **ssl** (*Required*): Enable or Disable SSL/TLS for the editor.
-- **certfile** (*Required*): Set the path the your SSL certificate if the ssl-option is set to `true`.
-- **keyfile** (*Required*): Set the path the your SSL private key if the ssl-option is set to `true`.
-- **allowed_networks** (*Required*): Limit access to the configurator by adding allowed IP addresses/networks to the list.
-- **banned_ips** (*Required*): List of statically banned IP addresses.
-- **banlimit** (*Required*): Ban access from IPs after `banlimit` failed login attempts. The default value `0` disables this feature. Restart the add-on to clear the list of banned IP addresses.
-- **ignore_pattern** (*Required*): Files and folders to ignore in the UI.
-- **dirsfirst** (*Required*): List directories before files in the file browser.
-- **enforce_basepath** (*Required*): If set to `true`, access is limited to files within the `/config` directory.
-- **notify_service** (*Required*): Specify a custom notify-service to be used to push notifications.
-- **ignore_ssl** (*Required*): Ignore SSL errors when accessing the Home Assistant API.
-- **sesame** (*Optional*): Secret token to dynamically allow access from the IP the request originates from. Open your bookmark https://hassio.yourdomain.com:8123/somesecretnobodycanguess while `allowed_networks` is set to `[]` and your IP will get whitelisted. You can use the _Network status_ menu to revoke IP addresses for which access has been granted. Regular authentication is still required.
-- **sesame_totp_secret** (*Optional*): Like the `sesame` option, but instead as Base32 encoded secret string must be provided. This string then can be added to a TOTP App like Google Authenticator. This way you get a 6-digit `sesame` that changes every 30 seconds.
-- **loglevel** (*Optional*): You can change the logging level from the default value `info` if you want to. Valid values are: `debug`, `info`, `warning`, `error`, `critical`.
+{% configuration %}
+username:
+  description: Set a username so that access your configuration is protected.
+  required: true
+  type: string
+password:
+  description: Set a password for access.
+  required: true
+  type: string
+ssl:
+  description: Enable or Disable SSL/TLS for the editor.
+  required: true
+  type: string
+certfile:
+  description: Set the path the your SSL certificate if the ssl-option is set to true.
```

Comment:
Please use markup as per the original

Reply:
This *IS* what was written in the original....

```
- **certfile** (*Required*): Set the path the your SSL certificate if the ssl-option is set to `true`.
```

Reply:
I see what you mean, fix submitted.